jsonld/examples/package_sbom.json

```diff
@@ -0,0 +1,79 @@
+{
+    "@context": "https://raw.githubusercontent.com/spdx/spdx-3-serialization-prototype-playground/main/jsonld/spdx-3.0-context.json-ld",
+    "@graph": [
+        {
+            "type": "CreationInfo",
+            "spdxId": "_:creationinfo",
+            "createdBy": "http://spdx.example.com/Agent/JoshuaWatt",
+            "specVersion": "3.0.0",
+            "created": "2024-03-06T00:00:00Z"
+        },
+        {
+            "type": "Person",
+            "spdxId": "http://spdx.example.com/Agent/JoshuaWatt",
+            "name": "Joshua Watt",
+            "creationInfo": "_:creationinfo",
+            "externalIdentifier": [
+                {
+                    "type": "ExternalIdentifier",
```

Comment:
Isn't an `@id` missing here?

Reply:
No, it's an implict blank node. I would highly recommend against requiring an @id here.

Reply:
I think we have said that _every_ JSON object will have an `@id`. It might be never be referenced, but it has to exist.

Reply:
My understanding is that every Element needs a `spdxId` (although, again I don't know how you would actually enforce that, but it's a different story), but that non-Element objects do not need one (e.g. a blank node is fine). I think this is likely doubly true if non-Elements are supposed to use @id and can be assigned a blank-node id by the user anyway.

I would recommend against requiring an @id on non-Element classes unless there is some fantastically good reason for it that I'm unaware of; do you happen to recall the history of that?